src/SignalR/clients/java/signalr/src/test/java/com/microsoft/signalr/JsonHubProtocolTest.java

```diff
@@ -5,13 +5,12 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import java.util.ArrayList;
-import java.util.Arrays;
+import java.lang.reflect.Type;
```

Comment:
We need to add some tests to Json now for more complex types right?

Reply:
Yeah, I'll whip up a few